app/views/data_access/_access_error.html.erb

```diff
@@ -1,6 +1,14 @@
 <div>
-  <p><strong>Error: </strong> <span data-bind="text: error_code"></span></p>
-  <p><strong>Message: </strong> <span data-bind="text: error_message"></span></p>
+  <p>
+    <strong>Error: </strong>
+    <span data-bind="text: error_code"></span>
+  </p>
+  <p>
+    <strong>Message: </strong>
+    <ul data-bind="foreach: error_message">
+      <li data-bind="text: $data"></li>
+    </ul>
```

Comment:
@mreese84 This is what this looks like with multiple errors. Even with one error it puts the message below the `Message:` text. How can we make that look better?

![screen shot 2016-08-22 at 1 12 34 pm](https://cloud.githubusercontent.com/assets/171388/17864078/6ad8dbbc-686a-11e6-9508-18ae79a08591.png)